client/src/components/Workflow/List/WorkflowList.vue

```diff
@@ -18,14 +20,20 @@ import FilterMenu from "@/components/Common/FilterMenu.vue";
 import Heading from "@/components/Common/Heading.vue";
 import ListHeader from "@/components/Common/ListHeader.vue";
 import LoginRequired from "@/components/Common/LoginRequired.vue";
+import TagsSelectionDialog from "@/components/Common/TagsSelectionDialog.vue";
 import LoadingSpan from "@/components/LoadingSpan.vue";
 import WorkflowListActions from "@/components/Workflow/List/WorkflowListActions.vue";
 
-library.add(faStar, faTrash);
-
 type ListView = "grid" | "list";
 type WorkflowsList = Record<string, never>[];
 
+// Temporary interface to match the `Workflow` interface from `WorkflowCard`
```

Comment:
```suggestion
// Interface to match the `Workflow` interface from `WorkflowCard`
```
If you're not saying what needs to be adopted in which way this is going to cause confusion to the next person looking at it.